client/src/App.jsx

```diff
@@ -1,59 +1,205 @@
 
-import React, {Component} from 'react'
+import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
-import Calendar from 'react-calendar'
+import Calendar from './Calendar.jsx';
+import ReservationSummary from './ReservationSummary.jsx';
+import $ from 'jquery';
 
-import DateRangePicker from 'react-daterange-picker'
-//import 'react-daterange-picker/dist/css/react-calendar.css'
-//import DateRangePicker from '@wojtekmaj/react-daterange-picker'
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      date: new Date()
+      dates: [],
+      checkIn: 'notSelected',
+      checkOut: 'notSelected',
+      showing: false, //is calendar showing
```

Comment:
Great work on creating the necessary state variables!!
However, it is worth exploring naming convention of Boolean variables.
For eg: 'showing' could be named 'isCalendarShowing'